xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/layouts/table/LayoutTableHeaderFilters.vue

```diff
@@ -80,7 +80,8 @@ export default {
 <style>
 
 .layout-table .column-filters th {
-  padding: 2px;
+  padding: 0px;
+  border: none;
```

Comment:
We should keep the border in order to preserve the standard XWiki table styles. The table layout should look as much as possible like any other table inside XWiki. Moreover, I think it looks better if the header and the filter row are visually separated from the table body with this thin border.